dataprofiler/labelers/base_model.py

```diff
@@ -5,10 +5,12 @@
 import copy
 import inspect
 import warnings
-from typing import Any, Dict, Iterator, List, Optional, Tuple, Type, Union
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Tuple, Type, Union
 
-import numpy as np
-import pandas as pd
+if TYPE_CHECKING:
+    import tensorflow as tf
```

Comment:
This i'm concerned about, what happens if someone doesn't have TF installed?

This is why we put it in the class itself to avoid the req unless executed. I understand this won't hit at runtime, but I guess someone could operate on the code w/o TF?

Reply:
Actually, `line 53` shouldn't not be a `tf.keras.model` truly it is `Any`. So we shouldn't need this line anyways?

Reply:
I changed it